ts/examples/node-examples/src/combine_mp4.tsx

```diff
@@ -0,0 +1,90 @@
+import { OfflineCompositor } from '@live-compositor/node';
+import {
+  View,
+  Text,
+  Tiles,
+  Rescaler,
+  InputStream,
+  useCurrentTimestamp,
+  useAfterTimestamp,
+  Show,
+} from 'live-compositor';
+import { downloadAllAssets } from './utils';
+import path from 'path';
+import { useState } from 'react';
+
+function ExampleApp() {
+  return (
+    <Tiles transition={{ durationMs: 200 }}>
+      <InputTile inputId="input_1" />
+      <Show delayMs={2000}>
+        <InputTile inputId="input_2" />
+      </Show>
+      <Show timestampMs={{ start: 5000, end: 8000 }}>
```

Comment:
either `<Show timeRange={{ start: 5000, end: 8000 }}>` or `<Show timeRangeMs={{ start: 5000, end: 8000 }}>`, but I prefer the first one - I'm not sure if that many libraries specify the unit in the props in this manner. You can usually safely assume that it's ms.